Utilities/ReleaseScripts/python/cmsCodeRules/config.py

```diff
@@ -63,7 +63,7 @@
 
 Configuration[ruleName]['description'] = 'Search for "catch(...)" statements in *.cc, *.cxx files'
 Configuration[ruleName]['filesToMatch'] = ['*.cc', '*.cxx']
-Configuration[ruleName]['exceptPaths'] = ['FWCore/*', 'EventFilter/*', '*/*/test/*', '*/*/bin/*']
+Configuration[ruleName]['exceptPaths'] = ['FWCore/*', 'EventFilter/*', 'HeterogeneousCore/*', '*/*/test/*', '*/*/bin/*']
```

Comment:
By quick look we no longer have any `catch(...)` inside `HeterogeneousCore`. If there is no other reason, could this addition be removed?

Reply:
We have this:
https://github.com/cms-sw/cmssw/blob/6dcf685e9f10cc422cb50b1973fe156b12f48494/HeterogeneousCore/CUDACore/src/ScopedContext.cc#L27-L34

If the rule checker does not complain, I can drop the exception.

Reply:
As far as I can tell the check is for `catch (...)` and `catch (cms::Exception&)` should be fine.

Reply:
Ah - I see.
OK, that is in the main releases since a while, so if there were no complaints, we are fine.